freqtrade/configuration.py

```diff
@@ -0,0 +1,199 @@
+"""
+This module contains the configuration class
+"""
+
+import json
+
+from typing import Dict, List, Any
+from jsonschema import Draft4Validator, validate
+from jsonschema.exceptions import ValidationError, best_match
+
+from freqtrade.constants import Constants
+from freqtrade.logger import Logger
+
+
+class Configuration(object):
```

Comment:
Do you think it makes sense to implement `__getitem__()` for `Configuration`?

That would give us a convinient way to access the config without `get_config()`, e.g:
```
conf = Configuration(...)
dry_run = conf['dry_run']
```